packages/backend/src/assets/ai.json

```diff
@@ -69,6 +69,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-java",
```

Comment:
perhaps do we want to specify it is quarkus-based, as we will probably add other java frameworks

Reply:
I personally would prefer simple names to not crowd the UI in podman desktop catalog view
yeah I was wondering same thing; but as the list in here just says "chatbot" without mentioning python nor its streamlit based .... so the biggest diff which in this case is "java" seemed good balance.

maybe `chatbot-java-quarkus`, `chatbot-python-streamlit`, `chatbot-python-flask` etc. is a good pattern to apply?